tools/mksurfdata_map/src/mkfileMod.F90

```diff
@@ -553,6 +553,10 @@ subroutine mkfile(domain, fname, harvdata, dynlanduse)
                long_name=mkharvest_longname(ind2D(j)), units=mkharvest_units(ind2D(j)) )
        end do
        deallocate(ind1D, ind2D)
+       
+       ! IV: add lake field in here? 
```

Comment:
Can remove the comment. It's clear it needs to be on the landuse.timeseries file.

Reply:
I have removed this in a cleanup commit that I will have @Ivanderkelen merge into this branch.